app/assets/stylesheets/admin.scss

```diff
@@ -188,3 +188,35 @@ label {
 .data_update_script__status {
   min-width: 120px;
 }
+
+// Navbar links have text-decorations that are being
+// rendered from boostrap. We want to remove these to be
+// consistent with links in the crayons design system.
+.admin__left-sidebar {
+  .crayons-link {
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: none;
+    }
+  }
+
+  button {
+    border: 0;
+    width: 100%;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0;
+  }
+
+  :focus-visible {
+    outline: 2px auto blue;
+  }
+
+  :focus:not(:focus-visible) {
+    outline: none;
+  }
```

Comment:
It seems focus-visible is not supported in Safari. Looking for a workaround.

Reply:
looks like this should work:

```scss
.crayons-link {
    text-decoration: none;

    &:focus {
      outline: 2px auto blue;
    }

    &:hover {
      text-decoration: none;
    }
  }
```

Reply:
This article was great in helping me to figure out the issue. https://matthiasott.com/notes/focus-visible-is-here: Using progressive enhancement we can define regular focus styles for non-supporting browsers and then overwrite them for browsers that support :focus-visible.

Reply:
or rather I mean move all the focus styles into here, to avoid seeing that focus indicator on click too - 

```scss
.crayons-link {
    text-decoration: none;

    &:focus {
      outline: 2px auto blue;
    }

    &:hover {
      text-decoration: none;
    }

    &:focus:not(:focus-visible) {
      outline: none;
    }
  }
```

Reply:
Thanks @aitchiss! Done and pushed ✅ 😊